client/extensions/woocommerce/state/sites/settings/email/README.md

```diff
@@ -0,0 +1,106 @@
+Email Settings
+================
+
+This module is used to manage settings for products for a site.
+
+## Actions
+
+### `fetchEmailSettings( siteId: number )`
+
+Pull email settings from the remote site. Does not run if the settings are loading or already loaded.
+
+## Reducer
+
+This is saved on a per-site basis, either as "LOADING" (when requesting settings), or a list of settings as returned from the site's `settings_email_groups` API.
+
+```js
+{
+	"settings": {
+		"email": "LOADING",
+	}
+	// or
+	"settings": {
+		"email": [
```

Comment:
Thank you for that comment. I see how it my be confusing. I have added additional explanation. Please tell me if it helps - if not I will gladly update it again.

Reply:
The explanation helps, thanks! I wonder if putting `group_id` first and then `id` and `value` after it would help, too. That way it's clear that `group_id` is the primary qualifier.